ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDInterestDateType.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+public enum CCDInterestDateType {
```

Comment:
why don't we extract this enum `uk.gov.hmcts.cmc.domain.models.InterestDate.InterestDateType` are reuse it here?

Reply:
they are based on ccd definition excel sheet